system/include/emscripten/console.h

```diff
@@ -20,6 +20,14 @@ void emscripten_console_logf(const char *utf8String, ...) __attribute__((__forma
 void emscripten_console_warnf(const char *utf8String, ...) __attribute__((__format__(printf, 1, 2)));
 void emscripten_console_errorf(const char *utf8String, ...)__attribute__((__format__(printf, 1, 2)));
 
+// Write to the out() and err() hooks directly (defined in shell.js).
+// These have different behaviour compared to console.log/err.
+// Under node, they write to stdout and stderr which is a more direct way to
+// write output especially in worker threads.
+// See https://github.com/emscripten-core/emscripten/issues/14804
```

Comment:
They are also aware of `Module.print / Module.printErr`, which is worth mentioning too. Given that I would basically describe them as writing to the standard out or standard error streams, including user overrides.

Reply:
Yes, it I agree its worth mentioning the module overrides.   How about adding one more sentence such as:  "The default behavior of these functions can be overridden by `print` and `printErr`, if provided on the Module object."